src/main/java/teammates/common/util/Const.java

```diff
@@ -1193,6 +1193,9 @@ private void _______values_with_special_meanings________________________(){}
     public static final Date TIME_REPRESENTS_LATER;
     public static final Date TIME_REPRESENTS_NOW;
     
+    //represents time (in milliseconds) from epoch (1/1/1970) to default student creation date (1/1/2011) in UTC
+    public static final long DEFAULT_STUDENT_CREATION_DATE = 1293840060000L;
```

Comment:
Even if explained, this number looks "magic". Consider parsing a date string to a Date object and then getting the long value from it.